src/main/kotlin/pt/up/fe/ni/website/backend/service/ActivityService.kt

```diff
@@ -16,6 +16,10 @@ abstract class ActivityService<T : Activity>(
     fun addTeamMemberById(idActivity: Long, idAccount: Long): T {
         val activity = getActivityById(idActivity)
         val account = accountService.getAccountById(idAccount)
+        if (activity.hallOfFame.find { it.id == idAccount } != null) {
+            // TODO: deal with member that already exists on the hallOfFame, should it return an error or move?
+            return moveMemberToActiveTeamById(idActivity, idAccount)
```

Comment:
I think the logic makes sense but it **needs** to be documented in the API, since it's not straight forward